code/controllers/subsystem/init/xenoarch.dm

```diff
@@ -15,6 +15,7 @@ var/datum/subsystem/xenoarch/SSxenoarch
 
 /datum/subsystem/xenoarch/Initialize(timeofday)
 	SetupXenoarch()
+	initialized = TRUE
```

Comment:
```suggestion
	..()
```
unless there's an actual reason for doing it your way

Reply:
There isn't really, I'm just scared of breaking anything if I call the parent

Reply:
but I guess there's no basis for that.